tools/autograd/gen_python_functions.py

```diff
@@ -31,6 +31,11 @@
     'max_pool1d', 'max_pool2d', 'max_pool3d'
 ]
 
+NEVER_SKIP = {
```

Comment:
What does this do? Needs a comment.

Reply:
Can we please not complicate the codegen with another special case set? How about changing the regex above to be `'((?!_thnn_fused_(rnn|lstm)_cell).)*_backward'` and leave a comment there?

Reply:
Maybe it's a matter of taste, but for me adding this set is less complicated than using a regex like that (and scales much better). Can change it though if people feel strongly about it.

Reply:
Hah, good news is that I realized this change is not longer necessary! I needed the backward function when I was calling it from the `backward` of the Python `Function`, but now that everything happens via autograd in C++, this is no longer needed!